test/tomo/operator/xray_trafo_test.py

```diff
@@ -30,28 +30,261 @@
 # Internal
 import odl
 from odl.util.testutils import almost_equal
+from odl.tomo import ASTRA_CUDA_AVAILABLE
+from odl.tomo.util.testutils import skip_if_no_astra_cuda
 
 
-@pytest.mark.xfail  # Expected to fail since scaling of adjoint is wrong.
-@pytest.mark.skipif("not odl.tomo.ASTRA_CUDA_AVAILABLE")
-def test_xray_trafo():
-    """Test discrete X-ray transformt using ASTRA with CUDA."""
+# TODO: increase test coverage and improve tests
```

Comment:
Look into the largescale tests, we have pretty much exactly this except we do tomography at the end. Simply change that part to an adjoint check and you should be fine?